File: VehicleServiceCenterNew/BackEnd/src/main/java/lk/vsc/service/LoginService.java
package lk.vsc.service;

import lk.vsc.entity.Employee;
import lk.vsc.entity.Login;
import lk.vsc.entity.Vehicle;

public interface LoginService {
    

    String loginFunction(String nic, String password);

    Login addLoginDetails(Login login);
}

File: VehicleServiceCenterNew/FrontEnd/src/app/Service/login.service.ts
import { Injectable } from '@angular/core';
import {Employee} from "../Model/Employee";
import {Vehicle} from "../Model/Vehicle";
import {environment} from "../../environments/environment";
import {HttpClient} from "@angular/common/http";


const URL ='/LoginController';
@Injectable({
  providedIn: 'root'
})


export class LoginService {

  public constructor(private http :HttpClient) { }

    loginFucntion(emp: Employee) {
    
        return this.http.get<String>(environment.backend_url + URL + '/Login/'+emp.nic+'/'+emp.password);

    }
}
